lib/portlayer/exec2/id.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exec2
+
+import "github.com/docker/docker/pkg/stringid"
+
+// ID is a container or process unique id
+type ID string
+
+// NilID is a placeholder for an empty container ID
+const NilID ID = ID("")
```

Comment:
K. I'll go with the suggestion to move to the Google UUID package. Good style tip though